Update the website content (#459)
pulp/pulpproject.org: content-plugins.md

## Pulp 3 CLI Coverage

The [Pulp 3 CLI](https://github.com/pulp/pulp-cli) continuosly gets expanded.
The CLI is implemented by the plugin writers, so the workflows that are possible with the CLI vary
from plugin to plugin.
Please check the CLI directly to see the current workflow coverage for the desired plugin.

## Role Based Access Control Support

While the functionality is available in Pulpcore, each plugin must add it separately.
Please check whether the plugin of your interest has already added RBAC.

## Cloud Storage

It is possible to configure Pulp to use [cloud
storage](https://docs.pulpproject.org/pulpcore/installation/storage.html). However, plugins can

> introduce changes that are incompatible with, for example, S3 requirements. A numbner of plugins

Reviewer: ```suggestion
introduce changes that are incompatible with, for example, S3 requirements. A number of plugins
```

> are regularly tested to ensure they remain compatible but the level of coverage might not cover all

Reviewer: ```suggestion
are regularly tested to ensure they remain compatible but the level of coverage might not cover our entire matrix of plugins.
```

> plugins matrix we have.

Reviewer: ```suggestion

```

## Pulp 3 CLI Coverage

> The [Pulp 3 CLI](https://github.com/pulp/pulp-cli) continuosly gets expanded.

Reviewer: ```suggestion
The [Pulp 3 CLI](https://github.com/pulp/pulp-cli) is continuously being expanded.
```

The CLI is implemented by the plugin writers, so the workflows that are possible with the CLI vary
from plugin to plugin.
Please check the CLI directly to see the current workflow coverage for the desired plugin.

## Role Based Access Control Support

While the functionality is available in Pulpcore, each plugin must add it separately.

> Please check whether the plugin of your interest has already added RBAC.

Reviewer: ```suggestion
Please check whether the plugin of your interest has already added RBAC support.
```

## Cloud Storage

It is possible to configure Pulp to use 
[cloud storage](https://docs.pulpproject.org/pulpcore/installation/storage.html). However, plugins
can introduce changes that are incompatible with, for example, S3 requirements. A number of plugins
are regularly tested to ensure they remain compatible but the level of coverage might not cover our
entire matrix of plugins.

## Pulp 3 Content Plugin Features

> Here are some highlights of each plugin capabilities.

Reviewer: ```suggestion
Here are some highlights of each plugin's capabilities.
```

* Use Pulp as a pull through cache for Maven content.

#### Ruby Gem

* Synchronize remote repository content and metadata locally.
* Upload your own content.
* Publish content to one or more repositories.

### Unmaintained plugins

The following plugins have been created and developed to some extent but have not been regularly maintained.

> If you're interested in extending or maintaining the basic functionality of these plugins, [let us know](/get_involved/).

Reviewer: ```suggestion
If you're interested in extending or maintaining the basic functionality of these plugins, [let us know](/help/).
```

get_involved is gone now, yes? Is there a particular anchor (`## Contribute as a Developer` maybe?) Or should this point to `[developers guide](https://docs.pulpproject.org/contributing/index.html)` here?

Author: yes, you're right it should be `help`
